modules/icinga/manifests/config.pp

```diff
@@ -57,15 +57,12 @@
   }
 
   if $::aws_migration {
-
-    file { '/var/lib/icinga/log':
-      ensure => directory,
-      owner  => 'nagios',
-      group  => 'adm',
-      mode   => '2755',
+    @logrotate::conf { 'icinga':
+      matches => "/var/log/${title}/*.log",
```

Comment:
Where does `title` come from? Other apps seem to [name the directory explicitly](https://github.com/alphagov/govuk-puppet/blob/30b6c7aed5bcd73b95ca9ff9b1828416999de549/modules/nginx/manifests/logging.pp#L13-L14) (probably to avoid someone asking where `title` comes from!).

Reply:
Sure, it's gone 👍. I thought I knew where it came from, but now I'm not so sure.